indicators/facebook-appeal-lf46ph.yml

```diff
@@ -0,0 +1,38 @@
+title: Facebook Appeal Phishing Kit lf46pH
+description: |
+    Detects a phishing kit targeting Facebook by tricking users into filling out a fake Community Standards violation appeal form.
+
+
+references:
+    - https://urlscan.io/result/93543e89-756e-4baa-8201-72d75943b421/
+    - https://urlscan.io/result/bea1dc69-a55c-4d5d-a2bd-ca6da51bafa7/
+    - https://urlscan.io/result/b998e7ed-92e8-45ef-af4c-1f5fc38fb968/
+    - https://urlscan.io/result/cf35e2a2-e4cd-4ff3-a573-f900bfb18f95/
+    - https://urlscan.io/search/#hash%3Af467a3a908fdedcb8af1cfb80bed6f6e990b021945ad10247f2b94294ffcbb0f
+
+detection:
+
+    protection:
+      html|contains|all:
+        - function mousedwn(e)
+        - window.addEventListener("keydown",function(e)
+
+    title:
```

Comment:
Maybe the fake case number ID (`#1008501933214`) would be more worthwhile than the title of the page